siege_utilities/geo/gazetteers/wikidata_gazetteer.py

```
@@ -0,0 +1,314 @@
+    def __init__(
+        self,
+        *,
+        timeout: float = _DEFAULT_TIMEOUT,
+        cache_size: int = 1024,
+        sparql_url: str = _WIKIDATA_SPARQL_URL,
+        overpass_url: str = _OVERPASS_URL,
+    ) -> None:
+        if not REQUESTS_AVAILABLE:
+            raise ImportError(
+                "WikidataGazetteer requires requests."
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)